packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient.ts

```diff
@@ -812,7 +829,7 @@ class StatefulGraphChatListClient implements StatefulClient<GraphChatListClient>
       draft.status = 'creating server connections';
     });
     try {
-      this._notificationClient.subscribeToUserNotifications(userId);
+      void this._notificationClient.subscribeToUserNotifications(userId);
```

Comment:
no promise, why void?

Reply:
That came from eslint, but I think it was probably just because my code was out-of-date at the time. I removed it.